sample/src/main/java/io/pixsight/sample/scenemanager/SampleBindScenesActivity.kt

```diff
@@ -0,0 +1,70 @@
+package io.pixsight.sample.scenemanager
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import io.pixsight.scenemanager.SceneCreator
+import io.pixsight.scenemanager.SceneManager
+import io.pixsight.scenemanager.animations.SceneAnimations
+import io.pixsight.scenemanager.annotations.BuildScenes
+import io.pixsight.scenemanager.annotations.Scene
+import kotlinx.android.synthetic.main.activity_no_annotations_sample.*
+
+@BuildScenes(
+    value = [
+        Scene(scene = Scene.MAIN, viewIds = intArrayOf(
+            R.id.activity_no_annotations_sample_main_content,
+            R.id.activity_no_annotations_sample_main_content_another_view
+        )),
+        Scene(scene = Scene.SPINNER, viewIds = intArrayOf(
+            R.id.activity_no_annotations_sample_main_content_another_view,
+            R.id.activity_no_annotations_sample_loader
+
+        )),
+        Scene(scene = Scene.PLACEHOLDER, viewIds = intArrayOf(
+            R.id.activity_no_annotations_sample_placeholder
+        ))
+    ],
+    first = Scene.PLACEHOLDER
+)
+class SampleBindScenesActivity : AppCompatActivity(), View.OnClickListener {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_no_annotations_sample)
+        SceneManager.create(
+            SceneCreator.with(this)
+                .animation(SceneAnimations.TRANSLATE_X)
+        )
+
+        activity_no_annotations_sample_main_content.setOnClickListener(this)
```

Comment:
You did not use the same bindListeners pattern?

Reply:
non car il n'y a pas le inflateOnDemand a true sur cette activity